tests/local.mk

```diff
@@ -0,0 +1,57 @@
+
+kind_dir ?= $(PWD)/.kind
+kind_bin = $(go_bin)/kind
+
+crossplane_sentinel = $(kind_dir)/crossplane_sentinel
+registry_sentinel = $(kind_dir)/registry_sentinel
+
+cloudscale_version = $(shell yq -e '.parameters."pkg.appcat.provider.cloudscale".images.provider-cloudscale.tag' packages/provider/cloudscale.yml)
+exoscale_version = master
```

Comment:
I strongly suggest you merge some of the open renovate PRs in https://github.com/vshn/provider-exoscale/pulls, then make a new release, and finally pin the version here, before merging this PR.